app-backend/db/src/main/scala/com/azavea/rf/database/UserDao.scala

```diff
@@ -33,8 +33,13 @@ object UserDao extends Dao[User] {
     query.filter(fr"id = ${id}")
   }
 
-  def unsafeGetUserById(id: String): ConnectionIO[User] = {
-    filterById(id).select
+  def unsafeGetUserById(id: String, isOwn: Option[Boolean] = Some(true)): ConnectionIO[User] = isOwn match {
+    case Some(isOwn) if isOwn == true => filterById(id).select
```

Comment:
you can do this as `case Some(true)` and `case Some(false)`, which makes the cases look less similar and I think (if scala's pattern match is like haskell's at least) a tiny bit faster (no boolean condition evaluation to do on top of the pattern match)